oat-sa/tao-test-runner-qti-fe
# Feature/TR-1970/Preload extended text entry runtimes

Related to: https://oat-sa.atlassian.net/browse/TR-1070

Revamp the pre-caching of items to allow also preloading some interaction runtimes.

What has been done:
- conversion of the `itemPreloader` into ES6+
- refactoring of the `itemPeloader` to extract the asset preloaders and implement a common way to preload assets
- add an interaction preloader mechanism
- add a preloader for the extendedText interaction runtimes (CKEditor)

How to test:
- the unit tests must pass
- see the companion PR to see how to check the test runner
